src/ui/components/QAForm.tsx

```diff
@@ -1,35 +1,36 @@
 import React, { Component } from 'react';
 import IQuestion from '../../interfaces/IQuestion';
 import ICourse from '../../interfaces/ICourse';
+import IGrade from '../../interfaces/IGrade';
 import '../../styles/QAForm.less';
-import { get, post } from '../../services/api-service';
+import {
+  postQuestion,
+  getGradeList,
+  getCourseList,
+} from '../../services/api-service';
 import Dropdown, { Option } from 'react-dropdown';
 
 interface IState {
   courses: ICourse[];
   grades: IGrade[];
   formControls: {
-    email: Option;
+    userEmail: Option;
     course: Option;
     theme: Option;
     question: Option;
     grade: Option;
+    anon: boolean;
   };
 }
 
-interface IGrade {
-  id: number;
-  name: string;
-}
-
 export default class QAForm extends Component<{}, IState> {
```

Comment:
Kan skrive denne om til funksjonell statless

Reply:
Det var litt surr med det med den Dropdown komponenten jeg bruker mtp. state. Tror jeg burde lage en egen komponent da isåfall. Gjorde et lite forsøk istad, men det tok en del tid så bør lage en egen issue på å gjøre om komponenter til funksjonelle tenker jeg, slik at "vi har tid". I hvert fall for denne denne.